libs/oidc/common/src/lib/utils/web/jwt.util.ts

```diff
@@ -34,6 +34,6 @@ interface IDecoded {
   aud: string;
   iat: string;
   jti: string;
-  events: {};
+  events: object;
```

Comment:
Might causes issues downstream since it doesn't allow property value re-assignment nor can you add properties that the OG object does not have.

```js

interface ValidObject {
  [key: string] : boolean | string | number | ValidObject;
}

```